index.html

```diff
@@ -1,27 +1,32 @@
+<!DOCTYPE html>
 
-<html>
+<html lang="en">
 <head>
-	<meta charset="utf-8"><link rel="stylesheet" href="/main.css">	<title>Alia Borroho</title>
-	</head>
-	<body>
+    <meta charset="utf-8">
+
+    <title>Alia Borroho</title>
+    <link rel="stylesheet" href="/main.css">
```

Comment:
This sill look for a CSS file named `main.css` located at the root of our hard drive. This project has a `style.css` file next to the current file.